src/editors/multiselect.js

```diff
@@ -44,6 +53,10 @@ JSONEditor.defaults.editors.multiselect = JSONEditor.AbstractEditor.extend({
 
       for(i=0; i<this.option_keys.length; i++) {
         this.select_options[this.option_keys[i]] = this.input.children[i];
+	var label = this.select_titles[this.option_keys[i]];
+	if (label){
+	  this.input.children[i].text = label;
+	}
```

Comment:
There is already a shortcut method to set the titles of select options.  I'd prefer using that instead of recreating the logic here.

```js
this.theme.setSelectOptions: function(select_input, option_values, option_titles);
```